code/src/index.css

```diff
@@ -1,13 +1,183 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+@import url('https://fonts.googleapis.com/css2?family=Patrick+Hand+SC&display=block');
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  font-family: 'Patrick Hand SC', -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 14px;
+  letter-spacing: 1px;
+  background-color: #fdf3f3;
 }
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h1 {
+  font-size: 40px;
+  line-height: 38px;
+  font-weight: 600;
+  color: #542626;
+}
+#send-thought,
+#all-thoughts {
+  padding: 0;
+  margin: 0 auto;
+  position: relative;
+  width: 80%;
+  max-width: 600px;
+}
+#send-thought {
+  padding-top: 20px;
+  margin-bottom: 40px;
+}
+.form-send-thought {
+  position: relative;
+}
+.list-thoughts {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+}
+.thought {
+  position: relative;
+  background: #fff;
+  border-radius: 16px;
+  width: 100%;
+  list-style: none;
+  margin-bottom: 20px;
+  transition: all .3s ease;
+  overflow: hidden;
+  border: 1px solid rgb(255,137,137,0.2);
+}
+.thought-like-link {
+  padding: 30px;
+  display: block;
+  text-decoration: none;
+  color: #542626;
+}
+.thought:hover {
+  transform: scale(1.05);
+  box-shadow: 0px 5px 10px rgb(255,137,137,0.15);
+}
+.thought.liked {
+  background-color: #ffcbcb;
+  transform: scale(1.05);
+}
+.thought.liked a {
+  cursor: default;
+}
+.thought.liked a .thought-date {
+  color: #b77d7d;
+}
+.thought.liked a .thought-message {
+  color: #542626;
+}
+.thought-message {
+  font-size: 26px;
+  line-height: 28px;
+  font-weight: 600;
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.thought-date {
+  color: #bababa;
+  font-size: 16px;
+  display: block;
+  font-weight: 500;
+}
+.heart {
+  position: absolute;
+  border: none;
+  background: url(assets/heart-fill.svg) no-repeat;
```

Comment:
It's more safe to wrap actual url inside quotes - some browsers might have problem with understanding content without them :
```
background: url("assets/heart-fill.svg") no-repeat;
```